Sprint-3/revise/investigate/find.js

```diff
@@ -21,5 +21,11 @@ console.log(find("code your future", "z"));
 
 // a) How the index variable updates during the call to find
 // b) What is the if statement used to check
+
+//The if statement is used to determine or check if the string has the character that we are looking for 
 // c) Why is index++ being used?
+
+// I think it is used to locate the exact location of the character we will be looking for in our string.
```

Comment:
The code is designed to locate the given character in the given string, but the explanation you gave does not quite answer the questoin.

Can you also try asking ChatGPT the same question (after you feed it the code) so see how else one can answer this question?